vim/yanking-text-to-register.md

```diff
@@ -0,0 +1,21 @@
+# Yanking text to register
+
+Yanking and deleting text will save the text to the same unamed register. Which means every time you delete a word, your 
+yanked word will be overwritten. Which caused me endless amounts of frustration when I accidentally deleted a word and had to
+remember what I yanked earlier.
+
+I learned that you can specify a register to save your yanked word to by prepending your yank command with `"a`. Where a
```

Comment:
Thoughts on wrapping the `a` in backticks so it stands out? "Where a is a register" should probably be part of the previous sentence.

> I learned that you can specify a register to save your yanked word to by prepending your yank command with `"a`, where `a` is a register. There are ...